src/Compiler.ts

```diff
@@ -57,7 +111,7 @@ export function compileFilesWithOptions(fileNames: string[], options: CompilerOp
                 let outPath = sourceFile.fileName;
                 if (options.outDir !== options.rootDir) {
                     const relativeSourcePath = path.resolve(sourceFile.fileName)
-                                                   .replace(path.resolve(rootDir), "");
+                                                .replace(path.resolve(rootDir), "");
```

Comment:
You sure this should be gone? I like the old version better.

Reply:
auto format /shrug